src/app.ts

```diff
@@ -89,6 +96,31 @@ function createMailto(url: string): void {
   )
 }
 
+function addCustomCss(windowElement: BrowserWindow): void {
```

Comment:
I thought of a shorter alternative:

```ts
import * as utils from 'electron-util'

const platform = utils.platform({ macos: 'macos', linux: 'linux', windows: 'windows'})

function addCustomCSS(windowElement: BrowserWindow): void {
  windowElement.webContents.insertCSS(
    fs.readFileSync(path.join(__dirname, '..', 'css', 'style.css'), 'utf8')
  )
  
  const platformCSSFile = `style.${platform}.css`
  if (fs.existsSync(platformCSSFile)) {
    windowElement.webContents.insertCSS(
      fs.readFileSync(path.join(__dirname, '..', 'css', platformCSSFile), 'utf8')
    )
  }
}
```

We don't need to add any other CSS files if they don't have yet imo.